src/utils.ts

```diff
@@ -26,3 +26,7 @@ export async function exec(
 		return actions_exec.exec(commandLine, args, options);
 	});
 }
+
+export const lib = {
+	exec,
```

Comment:
I have a few comments on this:
* Is this a relevant change to the PR?
* What's the value add in importing everything as one module?
  Maybe down the line when there's multiple utility functions but I think it's better if utility functions are imported separately and not altogether.

If we're going to expose utilities as a single module then I think it makes sense to remove the `export` keyword from the `exec` function.

Reply:
My intention is to mock the exported function `exec`, and this exported const `lib` is a shim for that. Do you have any preferred approach for mocking? I can edit the pull request to accommodate that.

Reply:
@piraka9011 I'll let that up to you - could you merge the PR if you're ok with the current approach?

Reply:
I see, since we are using exec everywhere, wrapping it for a mock makes sense.
Thanks @seanyen!